lib/watir/window.rb

```diff
@@ -3,8 +3,14 @@ class Window
     include EventuallyPresent
     include Waitable
 
-    def initialize(driver, selector)
-      @driver = driver
+    def initialize(browser, selector)
+      if browser.is_a?(Watir::Browser)
+        @browser = browser
+        @driver = browser.driver
+      else
```

Comment:
boring :)

that's essentially private API, nobody uses it, just remove pointless conditions